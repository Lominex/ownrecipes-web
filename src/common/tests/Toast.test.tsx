import renderer from 'react-test-renderer';

import Toast from '../components/Toast';

test('Toast', () => {
  const component = renderer.create(
    <Toast
        show
        >
      Some string test
    </Toast>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
