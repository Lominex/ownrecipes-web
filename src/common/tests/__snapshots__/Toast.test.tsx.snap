// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast 1`] = `
<div
  aria-atomic="true"
  aria-live="assertive"
  className="fade toast simple-toast show"
  role="alert"
>
  <div
    className="toast-header"
  >
    <span
      className="toast-title"
    >
      Some string test
    </span>
  </div>
</div>
`;
